cppabi64/cppabi64.rst

```diff
@@ -0,0 +1,778 @@
+..
+   Copyright (c) 2003-2020, Arm Limited and its affiliates.  All rights reserved.
+   CC-BY-SA-4.0 AND Apache-Patent-License
+   See LICENSE file for details
+
+.. |release| replace:: 2021Q1
+.. |date-of-issue| replace:: 18\ :sup:`th` February 2021
+.. |copyright-date| replace:: 2003-2021
```

Comment:
this should read: 2010-2021

Reply:
How on earth did I manage that? I guess I must have been overenthusiastic about using CPPABI32 as a model for the overall document structure. Oops.